dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -37,8 +37,8 @@ ENV COMPILER="4.05.0"
 # Common OPAM packages.
 # `num` does not have a version number as the right version to install varies
 # with the compiler version.
-ENV BASE_OPAM="num ocamlfind.1.8.0 dune.1.6.2 ounit.2.0.8 odoc.1.4.0" \
-    CI_OPAM="menhir.20181113 elpi.1.4.0 ocamlgraph.1.8.8"
+ENV BASE_OPAM="num ocamlfind.1.8.0 dune.1.10.0 ounit.2.0.8 odoc.1.4.0" \
```

Comment:
Shouldn't we keep testing with the minimal required dune version?

Reply:
As long as we have the corresponding dune-lang we should be fine; there were some problems with old dune in 4.08.0 so IMHO it is more efficient to do like this.

Anyways we will require Dune 1.10 very soon, but yeah, we could go back on this.